dotcom-rendering/src/layouts/TagPageLayout.tsx

```diff
@@ -228,6 +232,11 @@ export const TagPageLayout = ({ tagPage, NAV }: Props) => {
 			</div>
 
 			<main data-layout="TagPageLayout" id="maincontent">
+				{isAccessibilityPage && (
+					<Island priority="enhancement" defer={{ until: 'visible' }}>
```

Comment:
I think this should be a `feature` or even `critical`, considering that it impacts accessibility and we never want to not render it should we adapt the page?